community/kernel/src/main/java/org/neo4j/kernel/impl/transaction/log/PhysicalLogFile.java

```diff
@@ -202,10 +202,21 @@ public static PhysicalLogVersionedStoreChannel openForVersion( PhysicalLogFiles
     {
         final File fileToOpen = logFiles.getLogFileForVersion( version );
 
-        if (!fileSystem.fileExists( fileToOpen ))
-            throw new FileNotFoundException(  );
+        if ( !fileSystem.fileExists( fileToOpen ) )
+        {
+            throw new FileNotFoundException( String.format( "File does not exist [%s]", fileToOpen.getCanonicalPath() ) );
+        }
+
+        StoreChannel rawChannel;
+        try
+        {
+            rawChannel = fileSystem.open( fileToOpen, "rw" );
+        }
+        catch ( FileNotFoundException e )
+        {
+            throw new FileNotFoundException( String.format( "File could not be opened [%s]", fileToOpen.getCanonicalPath() ) );
```

Comment:
Rather than wrapping, use `Exceptions.withCause` which merely extends the exception message.